swarm/api/manifest.go

```diff
@@ -34,6 +34,8 @@ import (
 const (
 	ManifestType        = "application/bzz-manifest+json"
 	ResourceContentType = "application/bzz-resource"
+
+	manifestSizeLimit = 100 * 1024 * 1024
```

Comment:
i think this is insanely big, since manifests are compacted in a trie, the limit can be much smaller